packages/UI/src/utils/formErrorScroller.tsx

```diff
@@ -0,0 +1,41 @@
+import type { FormikContextType } from 'formik';
+import { connect } from 'formik';
+
+type FormErrorScrollerProps = {
+  align?: ScrollIntoViewOptions['block'];
+  focusAfterScroll?: boolean;
+} & { formik?: FormikContextType<Record<string, unknown>> };
+
+const FormErrorScroller = ({
```

Comment:
@alicefate I made this component to automatically scroll to error input fields, 
usage: 
```jsx
import FormErrorScroller from '@devlaunchers/components/src/utils/formErrorScroller';
.
.
.
<FormErrorScroller focusAfterScroll /> {/* make sure to use this within formik rendering method */}
```
here's how we use it at recruitment app https://github.com/dev-launchers/dev-launchers-platform/pull/792/files#diff-8e96e73396d9633f0955f64672fe64175f662e854143af75c0daf2f1f54f36aa